src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -3878,6 +3885,69 @@ public override IOperation VisitTypeOf(ITypeOfOperation operation, int? captureI
             return new TypeOfExpression(operation.TypeOperand, semanticModel: null, operation.Syntax, operation.Type, operation.ConstantValue, operation.IsImplicit);
         }
 
+        public override IOperation VisitParameterInitializer(IParameterInitializerOperation operation, int? captureIdForResult)
+        {
+            var parameterRef = new ParameterReferenceExpression(operation.Parameter, semanticModel: null,
+                operation.Syntax, operation.Parameter.Type, constantValue: default, isImplicit: true);
+            VisitInitializer(rewrittenTarget: parameterRef, initializer: operation);
+            return null;
+        }
+
+        public override IOperation VisitFieldInitializer(IFieldInitializerOperation operation, int? captureIdForResult)
+        {
+            foreach (IFieldSymbol fieldSymbol in operation.InitializedFields)
+            {
+                IInstanceReferenceOperation instance = fieldSymbol.IsStatic ?
+                    null :
+                    new InstanceReferenceExpression(InstanceReferenceKind.ContainingTypeInstance, semanticModel: null,
+                        operation.Syntax, fieldSymbol.ContainingType, constantValue: default, isImplicit: true);
+                Optional<object> constantValue = fieldSymbol.HasConstantValue ? new Optional<object>(fieldSymbol.ConstantValue) : default;
```

Comment:
>Optional<object> constantValue = fieldSymbol.HasConstantValue ? new Optional<object>(fieldSymbol.ConstantValue) : default; [](start = 16, length = 122)

I don't think it is appropriate for the target to have a constant value. Here, we are actually simulating the initialization